includes/datavalues/SMW_DV_Number.php

```diff
@@ -407,8 +407,10 @@ public function getConvertedUnitValues() {
 	 * The returned value is a canonical ID for the main unit.
 	 * Returns the empty string if no unit is given for the value.
 	 * Overwritten by subclasses that support units.
+	 *
+	 * @return bool|string
 	 */
-	public function getUnit(): string {
+	public function getUnit() {
```

Comment:
This is a manual fix where Rector didn't understand that the subclasses return arrays.